<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="UsernameLabel2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RegisterUsernameLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RegisterPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorExistingLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;*Already Exisiting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorMatchingLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;*Not Matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RetypeRegisterPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RetypePasswordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="DateofBirthGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QComboBox" name="MonthComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DayLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="YearLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GenderGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="MaleRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FemaleRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccountTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="UserRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdminRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="ActionCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DramaCheckBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ComedyCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RomanceCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OtherCheckBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HorrorCheckBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="RegisterButton2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorCredentialsLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;*Please fill all credentials&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorAgeLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;*Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
